packages/pipeline/src/scripts/pull_missing_blocks.ts

```diff
@@ -37,7 +37,7 @@ interface MissingBlocksResponse {
 
 async function getAllMissingBlocksAsync(web3Source: Web3Source): Promise<void> {
     const blocksRepository = connection.getRepository(Block);
-    let fromBlock = EXCHANGE_START_BLOCK;
+    let fromBlock = 0;
```

Comment:
Do you need fromBlock as a parameter to the query? It looks like even without it, and even without sorting the blocks, you'd always pull MAX_BLOCKS_PER_QUERY new, distinct blocks. I'm sure sorting the blocks gives you a performance boost, but maybe you can just get rid of the fromBlock parameter entirely?

Reply:
👌 You're right. The `fromBlock` parameter is an artifact of an older implementation and we don't need it anymore. I removed it.